Content/Items/Vitric/Weapons.CoachGunUpgrade.cs

```diff
@@ -107,6 +119,80 @@ public override void ModifyShootStats(Player player, ref Vector2 position, ref V
 			}
 		}
 
+		public override void UseStyle(Player player, Rectangle heldItemFrame)
```

Comment:
Is there a degree to which we can abstract some of this animation? Between all the guns looks like alot of reuse of similar effects. Perhaps there is some virtue in a gun base class or helper function for doing these animations?